### DISCUSSION TITLE
How to Create sub menu in Module

### DISCUSSION THREAD

**Discussion by numanuddin1** [nasirkhan/laravel-starter#462] [General]
Hi, 
I want to create sub menus in module for example I have created Module name Configuration now i want sub menu in configuration like Configuration/Test1 and Configuration/Test2
Thanks

**Comment by nasirkhan** [comment]
You may create new modules and change the route for Test1 and Test2
modules.

Or

You may just create new controllers, routes and related files as needed.

There is no automated features available to make sub modules at this
moment.

--
*Nasir Khan Saikat*
nasirkhn.com

**Reply by numanuddin1** [reply]
Ok Thank you, 
Should i use modular approach or default directory for creating sidebar menus?

**Comment by nasirkhan** [comment]
Usually i create modules if the functionality of the module can be accessed from multiple places. Like I created `Tag` module, which is being used in `Post`/`Article` module. If the functionalities are not like that you may create a single module and add all the routes and controllers there. But you may use a different sub menu style as you like.
